doc/fd.1

```diff
@@ -376,10 +376,15 @@ parent directory
 path without file extension
 .IP {/.}
 basename without file extension
+.IP {{}
+literal '{'
 .RE
 
 If no placeholder is present, an implicit "{}" at the end is assumed.
 
+If you need to include the literal text of one of the placeholders, you can use "{{}" to
+escape the first "{". For example "{{}}" expands to "{}", and "{{}{{}}}" expands to "{{}".
```

Comment:
@tmccombs Thank you very much for working on this and for even providing two implementations.

If I'm being really honest, this paragraph sounds incredible confusing at first. It does make sense and is logically consistent, but it's.. certainly not easy to grasp. This is a quite complex escaping rule. And one that I haven't seen in use anywhere else.

Given that you now implemented this with a custom tokenizer, is there still no way to implement Rusts `{{` escaping or something like `\{`?

What does `parallel` do (we borrowed the `--exec` syntax from them)?

Reply:
oops. This branch actually does implement using `{{` and `}}` for escapes. I just forgot to update the help and documentation  to match :facepalm:.

Reply:
> What does `parallel` do (we borrowed the `--exec` syntax from them)?

I don't think you can escape braces in GNU parallel, but you can change the `{}` to a different string: https://stackoverflow.com/questions/42532693/